utils/preprocessing.py

```diff
@@ -40,7 +40,10 @@ def preprocess_co2_emissions(df: pd.DataFrame) -> pd.DataFrame:
     df.drop(['year', 'month', 'day'], axis=1, inplace=True)
     # To work with tonnes of co2 it is necessary a conversion factor of 3.664
     df['co2'] = df['co2'] / 3.664
-    df.rename(columns={'date': 'date', 'co2': 'co2_emissions'}, inplace=True)
+    df.rename(columns={'date': 'date', 'co2': 'value'}, inplace=True)
```

Comment:
I dont' agree with this. We can discuss it as a group in a meeting.

The virtue of having each dataframe with columns = ['date', name_of_the_predictor] is that we can call each predictor by its name directly. If we have only "value" columns, it's easier to make mistakes. I prefer to keep descriptive naming for the columns. And the name_of_the_predictor is the same that we are using in the config.